### PR TITLE
[lint-tool] Fix lint error in cordova tests

### PR DESCRIPTION
**Pull request by haoxli**
Replace tab space with whitespace using:
 grep "  " -rl * |grep ".html"|xargs -I% sed -i 's/    /    /g' %
 grep "  " -rl * |grep ".xml"|xargs -I% sed -i 's/    /    /g' %

https://crosswalk-project.org/jira/browse/CTS-1438

### CODE REVIEW

File: cordova/cordova-api-ios-tests/tests.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <test_definition>
   <suite category="Crosswalk Cordova" name="cordova-api-ios-tests">
     <set name="cordova" type="iosuiauto">
-    	<testcase component="Cordova Features/Mobile Spec" execution_type="auto" id="CrossWalk_Cordova_Mobile_Spec_Run" purpose="Validate 'cordova_mobile_spec' app can be run successfully">
```
Comment (Honry) [inline cordova/cordova-api-ios-tests/tests.xml]: Non-strict intent.

File: cordova/cordova-api-ios-tests/tests.full.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <test_definition>
   <suite category="Crosswalk Cordova" name="cordova-api-ios-tests">
     <set name="cordovaapi" type="iosuiauto">
-    	<testcase component="Cordova Features/Mobile Spec" execution_type="auto" id="CrossWalk_Cordova_Mobile_Spec_Run" priority="P0" purpose="Validate 'cordova_mobile_spec' app can be run successfully" status="approved" type="Functional">
+        <testcase component="Cordova Features/Mobile Spec" execution_type="auto" id="CrossWalk_Cordova_Mobile_Spec_Run" priority="P0" purpose="Validate 'cordova_mobile_spec' app can be run successfully" status="approved" type="Functional">
```
Comment (Honry) [inline cordova/cordova-api-ios-tests/tests.full.xml]: Non-strict intent.

File: cordova/cordova-api-ios-tests/tests.xml (hunk 1 above)
Comment (haoxli) [inline cordova/cordova-api-ios-tests/tests.xml]: Done

**Comment by Honry** [comment]
Thanks, LGTM.

**Pull request merged**